grails-app/controllers/org/broadinstitute/orsp/ApiInterceptor.groovy

```diff
@@ -10,7 +10,25 @@ import grails.converters.JSON
 class ApiInterceptor implements UserInfo  {
 
     ApiInterceptor() {
-        match controller: 'report', action: 'funding'
+        match controller: 'clarification'
+        match controller: 'dulNotify'
+        match controller: 'fileHelper'
+        match controller: 'infoLink'
+        match controller: 'issueReview'
+        match controller: 'project'
+        match controller: 'report'
+        match controller: 'sampleConsentLink'
+        match controller: 'consentGroup'
+        match controller: 'dataUse'
+        match controller: 'issueList'
+        match controller: 'history'
+        match controller: 'irb'
+        match controller: 'ne'
+        match controller: 'nhsr'
+        match controller: 'search'
+        match controller: 'status'
```

Comment:
I don't think this should be intercepted - it is an un-authed page that shows you this: http://orsp.broadinstitute.org/api/status

Reply:
And totally unrelated, the notification status looks like it has a bug
```
{
  "ok": false, 
  "systems": {
    "BSP": {
      "ok": true
    }, 
    "Crowd": {
      "ok": true
    }, 
    "DUOS": {
      "ok": true
    }, 
    "DataBioOntology": {
      "ok": true
    }, 
    "Database": {
      "ok": true
    }, 
    "GCS": {
      "ok": true
    }, 
    "Notifications": {
      "messages": [
        "No status available from Notifications Service"
      ], 
      "ok": false
    }, 
    "Ontology": {
      "ok": true
    }
  }
}
```